### ISSUE TITLE
Retraining SSD - Hollywood Heads Dataset

### ISSUE THREAD

**Issue by drmnasr** [dusty-nv/jetson-inference#742]
Hello,

I've been trying to retrain SSD following your tutorial but on the [Hollywood Heads Dataset](https://www.di.ens.fr/willow/research/headdetection/) I am getting a maximum of 72% accuracy with at currently 66 epochs, its a massive dataset but as you can see the loss from my training is as follows:

mb1-ssd-Epoch-0-Loss-2.846737966828959.pth
mb1-ssd-Epoch-1-Loss-2.755971641562949.pth
mb1-ssd-Epoch-2-Loss-2.762982871091478.pth
mb1-ssd-Epoch-3-Loss-2.6408235544695002.pth
mb1-ssd-Epoch-4-Loss-2.757811516429937.pth
mb1-ssd-Epoch-5-Loss-2.840624619053449.pth
mb1-ssd-Epoch-6-Loss-2.641128139809755.pth
mb1-ssd-Epoch-7-Loss-2.534017174968899.pth
mb1-ssd-Epoch-8-Loss-2.6970202052107424.pth
mb1-ssd-Epoch-9-Loss-2.6621399837601523.pth
mb1-ssd-Epoch-10-Loss-2.6127855282592174.pth
mb1-ssd-Epoch-11-Loss-2.6947037621538468.pth
mb1-ssd-Epoch-12-Loss-2.5016291404966275.pth
mb1-ssd-Epoch-13-Loss-2.5681323754750074.pth
mb1-ssd-Epoch-14-Loss-2.582128419584615.pth
mb1-ssd-Epoch-15-Loss-2.5853854298217915.pth
mb1-ssd-Epoch-16-Loss-2.6001623941813143.pth
mb1-ssd-Epoch-17-Loss-2.5413684180724583.pth
mb1-ssd-Epoch-18-Loss-2.5331809679542587.pth
mb1-ssd-Epoch-19-Loss-2.553739181506596.pth
mb1-ssd-Epoch-20-Loss-2.580593834382987.pth
mb1-ssd-Epoch-21-Loss-2.5550509524943315.pth
mb1-ssd-Epoch-22-Loss-2.479543300257955.pth
mb1-ssd-Epoch-23-Loss-2.547351667312993.pth
mb1-ssd-Epoch-24-Loss-2.6194528813085585.pth
mb1-ssd-Epoch-25-Loss-2.5789448000420596.pth
mb1-ssd-Epoch-26-Loss-2.622445685829862.pth
mb1-ssd-Epoch-27-Loss-2.5644632042575406.pth
mb1-ssd-Epoch-28-Loss-2.6049764825258883.pth
mb1-ssd-Epoch-29-Loss-2.4938448690882296.pth
mb1-ssd-Epoch-30-Loss-2.7207739212109376.pth
mb1-ssd-Epoch-31-Loss-2.6193668354641306.pth
mb1-ssd-Epoch-32-Loss-2.535341894925575.pth
mb1-ssd-Epoch-33-Loss-2.531399591402574.pth
mb1-ssd-Epoch-34-Loss-2.52521685970988.pth
mb1-ssd-Epoch-35-Loss-2.589461194870988.pth
mb1-ssd-Epoch-36-Loss-2.5785771697105657.pth
mb1-ssd-Epoch-37-Loss-2.5738729272142846.pth
mb1-ssd-Epoch-38-Loss-2.4956536037047456.pth
mb1-ssd-Epoch-39-Loss-2.6005861029542725.pth
mb1-ssd-Epoch-40-Loss-2.5449428892808275.pth
mb1-ssd-Epoch-41-Loss-2.5326706107507304.pth
mb1-ssd-Epoch-42-Loss-2.556398554935724.pth
mb1-ssd-Epoch-43-Loss-2.5182037403030457.pth
mb1-ssd-Epoch-44-Loss-2.5544695407619296.pth
mb1-ssd-Epoch-45-Loss-2.549499720054734.pth
mb1-ssd-Epoch-46-Loss-2.5622334486825338.pth
mb1-ssd-Epoch-47-Loss-2.5180420544827617.pth
mb1-ssd-Epoch-48-Loss-2.597714784190199.pth
mb1-ssd-Epoch-49-Loss-2.4625880037168724.pth
mb1-ssd-Epoch-50-Loss-2.4866301600462215.pth
mb1-ssd-Epoch-51-Loss-2.6740004199799325.pth
mb1-ssd-Epoch-52-Loss-2.7993546637434945.pth
mb1-ssd-Epoch-53-Loss-2.4584129157484886.pth
mb1-ssd-Epoch-54-Loss-2.503626000638292.pth
mb1-ssd-Epoch-55-Loss-2.5718448202251265.pth
mb1-ssd-Epoch-56-Loss-2.5624553135196244.pth
mb1-ssd-Epoch-57-Loss-2.468900358882444.pth
mb1-ssd-Epoch-58-Loss-2.5475869354409486.pth
mb1-ssd-Epoch-59-Loss-2.5277642962700893.pth
mb1-ssd-Epoch-60-Loss-2.5278560700461408.pth
mb1-ssd-Epoch-61-Loss-2.5451934245126.pth
mb1-ssd-Epoch-62-Loss-2.5374782263858937.pth
mb1-ssd-Epoch-63-Loss-2.4815356907986548.pth
mb1-ssd-Epoch-64-Loss-2.48723945236505.pth
mb1-ssd-Epoch-65-Loss-2.5604324659396864.pth
mb1-ssd-Epoch-66-Loss-2.4941094078614046.pth

i am using Learning Rate 0.0001, otherwise i didnt adjust any options. on the other hand i tried retraining with mb2-ssd-lite which yielded worse results. and if i ever dare to stop training and resume..the results are significantly worse (this goes for different datasets as well)

do you think i should keep the training going for hope of better accuracy or stop it? any advice would be really appreciated.

thank you.
